# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../spiralsynthmodular/nodef_val.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./SpiralSound/Plugins/Widgets/Fl_VU_Meter.cxx.org	Mon Jan 20 12:35:00 2003
+++ ./SpiralSound/Plugins/Widgets/Fl_VU_Meter.cxx	Mon Jan 20 12:35:42 2003
@@ -2,7 +2,7 @@
 #include "Fl_VU_Meter.h"
 #include <FL/fl_draw.H>
 
-Fl_VU_Meter::Fl_VU_Meter (int x, int y, int w, int h, const char *l = 0) :
+Fl_VU_Meter::Fl_VU_Meter (int x, int y, int w, int h, const char *l ) :
 Fl_Progress (x, y, w, h, l) {
 }
 
--- ./SpiralSound/Plugins/Widgets/Fl_VU_Meter.h.org	Mon Jan 20 12:35:05 2003
+++ ./SpiralSound/Plugins/Widgets/Fl_VU_Meter.h	Mon Jan 20 12:36:08 2003
@@ -5,7 +5,7 @@
 
 class Fl_VU_Meter : public Fl_Progress {
   public:
-    FL_EXPORT Fl_VU_Meter (int x, int y, int w, int h, const char *l = 0);
+    FL_EXPORT Fl_VU_Meter (int x, int y, int w, int h, const char *l );
   protected:
     FL_EXPORT virtual void draw();
 };
